java/src/jmri/jmrit/operations/locations/Track.java

```diff
@@ -786,8 +795,12 @@ public void setCommentBoth(String comment) {
             setDirtyAndFirePropertyChange("trackCommentBoth", old, comment); // NOI18N
         }
     }
-
+    
     public String getCommentBoth() {
+        return TrainCommon.getTextColorString(getCommentBothWithColor());
+    }
+
+    public String getCommentBothWithColor() {
         return _commentBoth;
     }
```

Comment:
What do these methods do? The javadoc for `TrainCommon.getTextColorString()` tells that it removes the text color from the string instead of returning the text color?